.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+<!--
+
+ PLEASE MAKE SURE NO SENSITIVE INFORMATION ABOUT YOUR
+ SYSTEM IS EXPOSED WHEN SUBMITTING FEEDBACK!
+
+ Repository Mirroring Tool (RMT) is an open-source project and we can make it
+ even better with your help and feedback.
+
+ When adding your feedback/bug please keep in mind:
+   - This is not a support channel. If you are having paid support from SUSE
+     business please go to https://scc.suse.com/support for support.
```

Comment:
I'd rephrase this as:

If you have paid support from SUSE, go to https://scc.suse.com/support to open a support ticket.